conversions/bytes_to_int.py

```diff
@@ -0,0 +1,67 @@
+def bytes_to_int(
+    bytes_var: bytes,
+    byteorder: str,
+    signed: bool,
+) -> int:
+
+    r"""
+    This script first converts a bytearray,
+    to its corresponding signed/unsigned binary number,
+    and then converts the binary number to an integer.
+
+    >>> tests = ((b"\x00\x10", "big", False), (b'\xfc\x00', "big", True),
+    ...          (b'\x00\x01', "big", False), (b'\x00\x10', 'little', False))
+    >>> all(bytes_to_int(a,b,c) == int.from_bytes(a,b,signed=c)
+    ...     for a,b,c in tests)
+    True
+    >>> bytes_to_int('abc',False,'big')
+    ValueError -  byteorder must be either 'little' or 'big'
+    0
+    >>> bytes_to_int(7.1, 'little', signed=True)
+    TypeError - 'float' object is not subscriptable
+    0
+
+    """
+
+    try:
+        if byteorder not in ("little", "big"):
+            raise ValueError(" byteorder must be either 'little' or 'big' ")
+
+        byteorder = byteorder.lower()
```

Comment:
Fix first, test second.
```suggestion
        byteorder = byteorder.lower()
        if byteorder not in ("little", "big"):
            raise ValueError(" byteorder must be either 'little' or 'big' ")
```

Reply:
Yes, my bad